corehq/apps/app_manager/xform.py

```diff
@@ -625,6 +630,13 @@ def audio_references(self):
     def video_references(self):
         return self.media_references(form="video")
 
+    def all_references(self, lang):
```

Comment:
Not sure about the performance of this. In the past I've found that the xpath find functions aren't that performant. Might be better to do a single pass and generate a map that gets memoized:
```python
media_ref_map = {}
for lang, translation_node in self.translations.items():
    media_ref_map[lang] = defaultdict(list)
    text_nodes = translation_node.findall('{f}text')
    for text in text_nodes:
        node = ItextNode(lang, text)
        for form, value in node.values_by_form.items():
            media_ref_map[lang][form].append(value.text)
```